src/app/3d/qgs3dnavigationwidget.cpp

```diff
@@ -196,11 +235,41 @@ Qgs3DNavigationWidget::Qgs3DNavigationWidget( Qgs3DMapCanvas *parent ) : QWidget
   gridLayout->addWidget( mMoveRightButton, 1, 3, 2, 1, Qt::AlignCenter );
   gridLayout->addWidget( mMoveDownButton, 3, 1, 1, 2, Qt::AlignCenter );
   gridLayout->addWidget( mMoveLeftButton, 1, 0, 2, 1, Qt::AlignCenter );
+
+  QHBoxLayout *layout = new QHBoxLayout;
+  layout->addWidget( mCameraInfo );
+  mCameraInfo->setVisible( false );
+
+  QCheckBox *cameraInfoCheckBox = new QCheckBox( this );
+  cameraInfoCheckBox->setText( "Show camera info" );
+  cameraInfoCheckBox->setChecked( false );
+  QObject::connect( cameraInfoCheckBox, &QCheckBox::stateChanged, parent,
+                    [ = ]( int state )
+  {
+    if ( state == Qt::CheckState::Checked )
+      mCameraInfo->setVisible( true );
+    else
+      mCameraInfo->setVisible( false );
+  }
+                  );
```

Comment:
I think you could simplify this connection by using `clicked(bool)` signal so the lambda function would just do `mCameraInfo->setVisible( value )`